packages/@aws-cdk/aws-ecs-patterns/lib/index.ts

```diff
@@ -1,3 +1,5 @@
+export * from './base/props';
```

Comment:
Do we really need this exported? The base interface e doesn’t need to be public

Reply:
Can you do this with JSII?

Reply:
OK, need to add `@internal` for this to work.

Reply:
What happens if you just not export this file here?

Reply:
`error TS9999: JSII: Unable to resolve referenced type '@aws-cdk/aws-ecs-patterns.BaseProps'. Type may be @internal or unexported`

Reply:
ahhh... apologies for the hassle... Let's revert this base struct... On second thought, it does not really make sense for these to be shared between the patterns...

Reply:
Sorry!